debug/src/debug.js

```diff
@@ -27,6 +27,16 @@ export function initDebug() {
 			);
 		}
 
+		Object.keys(vnode.props).forEach((key) => {
```

Comment:
Kind off afraid about the performance impact of this, since `Object.keys` iterates over the object and then we iterate over that created array. If this happens on every rerender...

Reply:
Well maybe you could do a inline `for(let key in vnode.props)` instead?

Reply:
Good point, will update this pr tomorrow after work

Reply:
It has been adjusted